examples/plot_extrapolation.py

```diff
@@ -52,7 +52,7 @@
 
 # Plot of diferent representations
 fig, ax = plt.subplots(2,2)
-fdgrid.plot(ax[0][0])
+fdgrid.plot(ax = [ax[0][0]])
```

Comment:
As of PEP8, named parameters are used without spaces:
```suggestion
fdgrid.plot(ax=[ax[0][0]])
```